src/Common/src/Interop/Windows/mincore/Interop.VersionHelpers.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Kernel32)]
+        private static extern bool VerifyVersionInfo([In] ref OSVERSIONINFOEX lpVersionInfo, uint dwTypeMask, ulong dwlConditionMask);
+
+        [DllImport(Libraries.Kernel32)]
+        private static extern ulong VerSetConditionMask(ulong dwlConditionMask, uint dwTypeBitMask, byte dwConditionMask);
```

Comment:
Haha, what i meant was, irrespective of the internal checks, when we return a value to the user, we said your system is Greater than or equal to Windows 8.1 for a Windows 10 device, which is true. But yeah, speaking of only the OS API, it lies. And it will lie, if using the VER_EQUAL mask. But there is also a way to make it not lie but setting the right parameter and returning a always true string.